### ISSUE TITLE
@inheritParams fails to inherit more than 1 argument from multi-argument definition lines

### ISSUE THREAD

**Issue by jefferis** [klutometis/roxygen#444]
e.g. base::merge or stats::heatmap 

```r
#' My merge
#'
#' @inheritParams base::merge
mymerge <- function(x, y) {}
```

Only y is documented (and CHECK will give a WARNING for missing x).

Looking at help for `base::merge`

```
x, y	data frames, or objects to be coerced to one.
```

So it seems that only the second (last?) variable listed is being picked up.

Tested with checkout of 7133fd62123eb3a8ea513a962d5c2f2d8d905da4

**Comment by jefferis** [comment]
I added a test in a88f986 which currently fails as below:

```
[snip]
Package documentation : ......
Param : ........12................
preref parsers : ............
[snip]

1. Failure (at test-param.R#69): @inheritParam can cope with multivariable argument definitions 
length(params) not equal to 2
1 - 2 == -1

2. Failure (at test-param.R#70): @inheritParam can cope with multivariable argument definitions 
sort(names(params)) not equal to c("x", "y")
Lengths (1, 2) differ
1 string mismatches:
y[2]: "y"
```

**Comment by jefferis** [comment]
fixed in PR #445
